packages/core/src/tracing/index.js

```diff
@@ -182,8 +183,14 @@ function initInstrumenations(_config) {
   if (!instrumenationsInitialized) {
     instrumentations.forEach(instrumentationKey => {
       instrumentationModules[instrumentationKey] = require(instrumentationKey);
-      const instrumentationName = instrumentationKey.match(/.\/instrumentation\/[^/]*\/(.*)/)[1];
-      const isInstrumentationDisabled = _config.tracing.disabledTracers.includes(instrumentationName.toLowerCase());
+
+      const extractedInstrumentationName = instrumentationKey.match(/.\/instrumentation\/[^/]*\/(.*)/)[1];
+
+      const isInstrumentationDisabled =
+        _config.tracing.disabledTracers.includes(extractedInstrumentationName.toLowerCase()) ||
+        (instrumentationModules[instrumentationKey].instrumentationName &&
+          _config.tracing.disabledTracers.includes(instrumentationModules[instrumentationKey].instrumentationName));
+
```

Comment:
_suggestion:_ We should have probable done that before, but how about extracting this part into a function that checks whether an instrumentation is disabled? It looks like we have the exact same snippet twice, once in `initInstrumenations` and once in `activate`.